palladio-graph-view.css

```diff
@@ -80,3 +80,59 @@
 [data-palladio-graph-view] .link.hidden {
     display: none;
 }
+
+.metrics {
```

Comment:
@jrladd Can you prefix all of these CSS class selectors with `[data-palladio-graph-view]`? If we don't do this, they could conflict with other classes in other components.